### ISSUE TITLE
Deliver the newsletter at a specified date, time.

### ISSUE THREAD

**Issue by jaceksynowiec** [contao/contao#6355]
Deliver the newsletter at a specified date, time.

You can now send a newsletter immediately, instantly.

It would be useful to define the date and time of the newsletter.
Plan your mailing in advance.

![contao-newsleeter-sending-time](https://cloud.githubusercontent.com/assets/51117/26752498/c39acc7a-4851-11e7-9f59-e68d776e6a2b.png)

**Comment by leofeyer** [comment]
I doubt that this will work, because a vital part of the sending process is the web browser, which executes the JavaScript that splits the process into several cycles and adds the necessary timeouts so the newsletter module does not flood the mail server. A scheduled sending would require a cron job, which has all these limitations.

**Comment by jaceksynowiec** [comment]
Could it be good to change the send from JavaScript to crone job?
This would give new opportunities.

**Comment by leofeyer** [comment]
> Could it be good to change the send from JavaScript to crone job?

No. Did you read my comment above?

**Comment by jaceksynowiec** [comment]
Oh yes.

So there are no solutions.
:(

**Comment by aschempp** [comment]
we *could* add a command to send newsletters, but actually I think the newsletter module is pretty limited and that's ok. For more features one should use a real newsletter platform.

**Comment by jaceksynowiec** [comment]
Would not it be useful to rebuild this system?

**Comment by jaceksynowiec** [comment]
**JavaScript is a strong limitation of the newsletter.**
Because you have to leave your browser after the post, and sending time is long ............... :(

It would be better if it was using PHP using Crone.
You can *Send* and close the browser.

**Comment by leofeyer** [comment]
Crone?

**Comment by jaceksynowiec** [comment]
Script PHP instead of JavaScript
+
Cron/crontab

**Comment by aschempp** [comment]
Sounds almost easy to do with our Messenger queue now @Toflar 😬

**Comment by Toflar** [comment]
I don't think that makes sense, this is better suited for a cronjob. But I have 0 reasons to work on the newsletter-bundle - haven't used it in over 10 years 😊
